nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -25,6 +25,33 @@
           <literal>services.nginx.virtualHosts.*.enableACME</literal>).
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>stdenv.mkDerivation</literal> now supports a
+          self-referencing <literal>finalAttrs:</literal> parameter
+          containing the final <literal>mkDerivation</literal> arguments
+          including overrides. <literal>drv.overrideAttrs</literal> now
+          supports two parameters
+          <literal>finalAttrs: previousAttrs:</literal>. This allows
+          packaging configuration to be overridden in a consistent
+          manner by providing an alternative to
+          <literal>rec {}</literal> syntax.
+        </para>
+        <para>
+          Additionally, <literal>passthru</literal> can now reference
+          <literal>finalAttrs.public</literal> containing the final
+          package, including attributes such as the output paths and
```

Comment:
Reading this now, I agree with @lilyball  think `package` or `finalPackage` would be a better name for this attribute.

Reply:
(would also help in functions like `testVersion`)